aws/resource_aws_ecs_task_definition.go

```diff
@@ -118,6 +118,7 @@ func validateAwsEcsTaskDefinitionNetworkMode(v interface{}, k string) (ws []stri
 	validTypes := map[string]struct{}{
 		"bridge": {},
 		"host":   {},
+		"awsvpc": {},
 		"none":   {},
```

Comment:
It's not necessarily a blocker for this PR, but I'm pondering with an idea of removing this validation completely as theoretically the list may grow again (knowing how complex and big the container networking "market" is). What do you think?

Reply:
In my opinion, it's better to prevent error as early as possible. When constructing state is better than when requesting API.
So I disagree with you.
Of course, I understood your concern but it won't release new network mode as soon as modifying codes coudn't follow it.

Reply:
My opinion comes from user respective but you have to consider maintainability more than me...
Finally I respect your decision👍